sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/flink-functions/arithmetic-functions.sqrl

```diff
@@ -0,0 +1,87 @@
+IMPORT flink.*; -- Import flink functions
+IMPORT data-local.Numbers; -- Import boolean data
+
+
+/* Create numeric data source */
+UnmodifiedNumericData := SELECT id,
+                                abs(avg(x)) AS x,
+                                abs(avg(y)) / 100 AS y,
+                                avg(z) AS z
+                         FROM Numbers GROUP BY id ORDER BY id;
+
+
+/* =======TEST CASES======== */
+
+/*+test */
+OperatorsTest := SELECT id,
+                         +x AS positive_operator,
+                         -x AS negative_operator,
+                         x + y AS addition_operator,
+                         x - y AS subtractions_operator,
+                         x * y AS multiplication_operator,
+                         x / y AS division_operator
+                         /*x % y AS mod_operator,*/
+                  FROM UnmodifiedNumericData
+                  ORDER BY id;
+
+/*+test */
+FunctionsTest := SELECT id,
+                        POWER(x, 3) AS power_function,
+                        ABS(x) AS absolute_function,
+                        /*SQRT(x) AS square_root_function,*/
+                        EXP(x) AS exponent_function
+                 FROM UnmodifiedNumericData;
+
+/*+test */
+LogarithmTest := SELECT id,
+                        LN(x) AS natural_log_function,
+                        LOG10(x) AS base_ten_log_function,
+                        LOG2(x) AS base_two_log_function
+                 FROM UnmodifiedNumericData;
```

Comment:
It could be helpful to have LOG(x) for when we map to other sql engines. The other log argument permutations can be omitted.